source/_posts/generating-pdf-in-android-using-itextpdf.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Generating Pdf the way you want, in Android using itextpdf
+date: 2020-08-23 05:45:58
+author: Ashish Jajoria
+categories:
+- ["Android"]
+tags: 
```

Comment:
NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping

Please put this keyword also in your blog content and tag

Reply:
Done